app/uk/gov/hmrc/exports/scheduler/jobs/emails/PagerDutyAlertManager.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.scheduler.jobs.emails
+
+import javax.inject.Inject
+import play.api.Logging
+import reactivemongo.api.ReadPreference
+import reactivemongo.bson.BSONObjectID
+import uk.gov.hmrc.exports.config.AppConfig
+import uk.gov.hmrc.exports.models.emails.SendEmailDetails
+import uk.gov.hmrc.exports.repositories.SendEmailWorkItemRepository
+import uk.gov.hmrc.workitem.WorkItem
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class PagerDutyAlertManager @Inject()(
```

Comment:
Cannot be a singleton?

Reply:
Could be. But I don't see much difference to be fair, but can make it Singleton if you think it is relevant.

Reply:
You're right here. I just think it's good practice. It does not prevent the approval anyway. Up to you at the end  :smile: